### PR TITLE
i18n(zh-cn): update configuration-reference.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Update configuration-reference.mdx by original English text.
- Uniform punctuation
- Fix something

#### Description

- Just doc

### CODE REVIEW

File: src/pages/zh-cn/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -301,14 +387,25 @@ export default defineConfig({
...
+### server.headers
+
+<p>
+
+**类型**：`OutgoingHttpHeaders`<br />
+**默认值**：`{}`<br />
+<Since v="1.7.0" />
+</p>
+
+设置要在 `astro dev` 和 `astro preview`` 中发送的自定义 HTTP 响应标头。
```
Comment (delucis) [inline src/pages/zh-cn/reference/configuration-reference.mdx:399]: Looks like an extra backtick snuck in here:

```suggestion
设置要在 `astro dev` 和 `astro preview` 中发送的自定义 HTTP 响应标头。
```

**Review by delucis — COMMENTED** [review]
Great work @liruifengv! Just spotted a couple of small things to fix but otherwise this is super helpful, thank you 🙌

File: src/pages/zh-cn/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -87,7 +87,7 @@ $ astro build --root ./my-project-directory
 <p>
 
 **类型**：`string`<br />
-**默认**：`"./dist"`
+**默认值**：`"./dist"`
 </p>
 
 设置 `astro build` 将你的最终构建写入的目录。
```
[the remark this answers is not in the document]
Comment (liruifengv) [reply src/pages/zh-cn/reference/configuration-reference.mdx:93]: Thank your review.

**Review by delucis — APPROVED** [review]
LGTM! 🚀

**Pull request merged**